### PR TITLE
[QPPSF-2593] 2018 Measure Spec Links

### PR DESCRIPTION
**Pull request by NateTheGreatt**
Added parsing for the measureSpecification column set with fields:

- default
- claims
- registry
- cmsWebInterface

Tech Spec:
[https://docs.google.com/document/d/1s4JkFWjVb3s6jUH1tteG6KKEjuy905nv63Ey5F6_g7k/edit?usp=sharing](
https://docs.google.com/document/d/1s4JkFWjVb3s6jUH1tteG6KKEjuy905nv63Ey5F6_g7k/edit?usp=sharing)

### CODE REVIEW

File: test/scripts/measures/2018/fixtures/test-quality-measures.csv [hunk 1]
```diff
@@ -66,11 +66,11 @@ Formula to determine the overall performance rate, given multiple strata of perf
...
+Coronary Artery Disease (CAD): Beta-Blocker Therapy - Prior Myocardial Infarction (MI) or Left Ventricular Systolic Dysfunction (LVEF < 40%),CMS145v6,N/A,70,7,Percentage of patients aged 18 years and older with a diagnosis of coronary artery disease seen within a 12-month period who also have a prior MI OR a current or prior LVEF <40% who were prescribed beta-blocker therapy,Effective Clinical Care,Process,Physician Consortium for Performance Improvement,-,-,X,-,-,X,-,-,X,-,-,-,-,X,X,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,multiPerformanceRate,2017,NULL,,FALSE,FALSE,NULL,2,2,weightedAverage,,https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Claims-Registry-Measures/2018_Measure_007_Registry.pdf,,,TRUE,FALSE
```
Comment (samskeller) [inline test/scripts/measures/2018/fixtures/test-quality-measures.csv]: Github is not being helpful here... but what changed in these lines? I don't see any differences...?

**Comment by sarahnw** [comment]
@samskeller the default field was/is for ACI/PI measures - we have links for them in 2018 but they're in a separate file since these are quality measures. I believe those are all blank right now in the repo. I have a separate task for PI updates we need to make. those specs will be in that update.

File: test/scripts/measures/2018/fixtures/test-quality-measures.csv (hunk 1 above)
Comment (NateTheGreatt) [reply test/scripts/measures/2018/fixtures/test-quality-measures.csv]: encoding difference I think, seems to be fixed now

**Comment by NateTheGreatt** [comment]
@samskeller  done deal, added those tests and updated the data

**Comment by samskeller** [comment]
Awesome, thanks @NateTheGreatt 🙏 appreciate the thorough PR! Merging now -- will prepare a release of this repo after

**Pull request merged**